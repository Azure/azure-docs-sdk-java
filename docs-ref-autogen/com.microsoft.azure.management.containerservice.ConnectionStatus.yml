### YamlMime:JavaType
uid: "com.microsoft.azure.management.containerservice.ConnectionStatus"
fullName: "com.microsoft.azure.management.containerservice.ConnectionStatus"
name: "ConnectionStatus"
nameWithType: "ConnectionStatus"
summary: "Defines values for ConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.microsoft.rest.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)"
- "com.microsoft.rest.ExpandableStringEnum.hashCode()"
- "com.microsoft.rest.ExpandableStringEnum.toString()"
- "com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class ConnectionStatus extends ExpandableStringEnum<ConnectionStatus>"
constructors:
- "com.microsoft.azure.management.containerservice.ConnectionStatus.ConnectionStatus()"
fields:
- "com.microsoft.azure.management.containerservice.ConnectionStatus.APPROVED"
- "com.microsoft.azure.management.containerservice.ConnectionStatus.DISCONNECTED"
- "com.microsoft.azure.management.containerservice.ConnectionStatus.PENDING"
- "com.microsoft.azure.management.containerservice.ConnectionStatus.REJECTED"
methods:
- "com.microsoft.azure.management.containerservice.ConnectionStatus.fromString(java.lang.String)"
- "com.microsoft.azure.management.containerservice.ConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.microsoft.azure.management.containerservice"
artifact: com.microsoft.azure:azure-mgmt-containerservice:1.38.1
