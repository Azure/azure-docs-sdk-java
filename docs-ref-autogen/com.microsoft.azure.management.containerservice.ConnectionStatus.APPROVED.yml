### YamlMime:JavaMember
uid: "com.microsoft.azure.management.containerservice.ConnectionStatus.APPROVED*"
fullName: "com.microsoft.azure.management.containerservice.ConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "ConnectionStatus.APPROVED"
members:
- uid: "com.microsoft.azure.management.containerservice.ConnectionStatus.APPROVED"
  fullName: "com.microsoft.azure.management.containerservice.ConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "ConnectionStatus.APPROVED"
  summary: "Static value Approved for ConnectionStatus."
  syntax: "public static final ConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.microsoft.azure.management.containerservice"
artifact: com.microsoft.azure:azure-mgmt-containerservice:1.38.1
